test/api/unit/libs/cron.test.js

```diff
@@ -1896,3 +1834,116 @@ describe('recoverCron', () => {
     }
   });
 });
+
+describe('cron for Dailies using every X days after completion', () => {
```

Comment:
If it's a new top level describe box it should go in a separate file, this one is already huge also

Reply:
No worries. You can ignore that commit though; it's a temporary one that I was using just to see if putting the new tests in their own box would help prevent timeouts (running the full test suite on my local install isn't usually reliable). I'm not expecting to keep the tests separated like that and I'm intending to move them back to where they were. 

I created it before I saw your comment about not using two or more clocks in one test and that idea is what I'm working on now.